main.go

```diff
@@ -43,35 +55,57 @@ func main() {
 			cmd.MarkFlagRequired("name")
 			cmd.ParseFlags(args)
 
-			argConf := config.Config{App: *app, Stack: *stack, Stage: *stage}
-			conf, err := config.Read(argConf, config.DefaultFiles()...)
-			check(logger, err, "Unable to read config", InvalidArgs)
-
-			ssm := store.NewSSM(logger, ssmClient(context.TODO(), logger, *profile))
+			check(logger, configErr, "Unable to read config", InvalidArgs)
 
-			service := store.Service{App: conf.App, Stack: conf.Stack, Stage: conf.Stage}
-			item, err := ssm.Get(service, *name)
+			item, err := ssmStore.Get(service, *name)
 			check(logger, err, fmt.Sprintf("unable to get %s for service '%s'", *name, service.Prefix()), 1)
 
 			logger.Infof(item.String())
 		},
 	}
 
+	getSecretCmd := &cobra.Command{
+		Use:   "get-secretmgr",
```

Comment:
As @kenoir suggests, I think we should consolidate the interface a bit. The aim of devx-config is to abstract over the actual storage of configuration both for better devx and also to allow us to change things in the future; by shaping the CLI so much to accommodate secrets manager, we break that.

What about either:

* supporting `list` access via a `--with-rotatable-secrets` bool flag on the pre-existing list command? (this would include ssm and sm parameters)
* or, in addition to above, also supporting `get`, `set`, `delete` via a `--is-rotatable-secret` bool flag? (and when setting for SM we should prompt that unless you want rotation (or cfn reference) SSM is to be preferred and get the user to confirm)

The reason I am slightly hesitant about supporting write/del for SM is that the SM interface is actually quite large so there is a chance of scope-creep over time or us just not serving it very well - i.e. the main feature of SM is rotation, but it's unclear how we could effectively support that via devx-config. So perhaps it is better to just support read operations here.

Most of the time we want to discourage use of secrets manager as it is more expensive than SSM. Rotation is normally the key benefit/difference (but unfortunately usage within Cloudformation is another).